app/server/crons/deactivateUsers.js

```diff
@@ -1,10 +1,8 @@
-SyncedCron.add({
-  name: 'Deactivate user',
-  schedule: function(parser) {
-    // parser is a later.parse object
-    return parser.text('every 1 day at 12:00 am');
-  },
-  job: function() {
+import { CronJob } from 'cron';
+
+new CronJob(
+  '0 0 12 * * *',
+  Meteor.bindEnvironment(function(err, data) {
     /* Get all users for whom the deactivation date is set */
     const users = Meteor.users
       .find({
```

Comment:
This query should probably filter out users who are already deactivated, i.e. where `isActive` is already `false`.

Reply:
Okay I think I can handle the update in only one query itself. Will change that.